ftplugin/python/SimpylFold.vim

```diff
@@ -116,6 +116,17 @@ function! s:NumContainingDefs(lnum)
 
 endfunction
 
+" Construct a foldexpr value
+function! s:FoldExpr(lnum, foldlevel)
```

Comment:
I don't think the name of this function elicits its purpose well. It doesn't construct a foldexpr value as stated; it separates two adjacent folds with the same fold level. Let's give it a more descriptive name.